packages/mui-material/src/NativeSelect/NativeSelectInput.js

```diff
@@ -35,10 +35,6 @@ export const StyledSelectSelect = styled('select')(({ theme }) => ({
     // Reset Chrome style
     borderRadius: 0,
   },
-  // Remove IE11 arrow
```

Comment:
Couldn't trace it further than https://github.com/mui/material-ui/pull/24698/files#diff-eca42ced51d2f5fc85b4932a8d16e28b815c6f193514d1f4101949e879970b12R53

Looks good in Chrome, Firefox, and Safari. Screenshot in Chrome:

<img width="260" alt="Screenshot 2024-05-23 at 16 45 49" src="https://github.com/mui/material-ui/assets/16889233/796b1746-1320-4945-9a8e-f39df3091204">

Reply:
#8976